source/docs/api-reference/api-usage/signal-logging.rst

```diff
@@ -211,3 +217,22 @@ An example error that may occur if the free space limit is reached is shown belo
 .. code-block:: text
 
    [phoenix] Signal Logger: Available disk space (3 MB) below 5 MB, stopping log
+
+Converting Signal Logs
+----------------------
+
+Signal logs can be converted to other common file formats such as WPILOG or MCAP using the :doc:`Tuner X Log Extractor </docs/tuner/tools/log-extractor>`.
+
+Additionally, the ``owlet`` CLI tool can be used from a terminal, including on platforms not supported by Tuner X. ``owlet`` can be downloaded from the `CLI Tools download page <https://docs.ctr-electronics.com/cli-tools>`__.
+
+To view a list of available commands, run ``owlet`` either with no parameters or with ``--help``.
+
+.. image:: images/owlet-cli.png
```

Comment:
I would prefer this to be a code block using "bash" instead of a picture, as it's easier to copy/paste and edit in the future.

Reply:
Wasn't really expecting us to need to edit it in the future, users should run the command to get the help message. I was just matching the caniv documentation.

Reply:
I think I would prefer doing this as well for the caniv documentation, but it's not a particular blocker on anything.